### PR TITLE
interface

### PR DESCRIPTION
**Pull request by Sindhujarajen**

### CODE REVIEW

File: first_project/src/app/goshopping/goshopping.component.ts [hunk 1]
```diff
@@ -1,35 +1,46 @@
...
-  // login(){
-  //   this.router.navigate(['/mat-table'])
-  // }
-  // goto(){
-  //   this.router.navigate(['/goshopping'])
-  // }
-  // wish(){
-  //   this.router.navigate(['/wishlist'])
-  // }
-}
```
Comment (Sushmitha-Vk) [inline first_project/src/app/goshopping/goshopping.component.ts:23]: comments

File: first_project/src/app/header/header.component.css [hunk 2]
```diff
@@ -31,12 +24,3 @@ div{
...
 }
-/* .search{
-      width: 100px;
-      margin-left: 100px;
-} */
-
-/* .example-full-width{ 
-    height: 80px;
-    margin-left:100px ;
-}  */
```
Comment (Sushmitha-Vk) [inline first_project/src/app/header/header.component.css:42]: comments

File: first_project/src/app/login/login.component.html [hunk 3]
```diff
@@ -1,46 +1,16 @@
...
-            <button  (click)="wish()">wishlist</button><br><br>
-            <button  (click)="logout()">logout</button>
-            </mat-drawer>
-     
-
-    <div class="example-sidenav-content">
-        <button type="button" mat-button (click)="drawer.toggle()">
-            menu
-        </button>
-        </div> -->
```
Comment (Sushmitha-Vk) [inline first_project/src/app/login/login.component.html]: comments

File: first_project/src/app/mat-table/mat-table.component.ts [hunk 4]
```diff
@@ -1,43 +1,29 @@
...
+import { Details } from '../model';
 
 @Component({
   selector: 'app-mat-table',
   templateUrl: './mat-table.component.html',
   styleUrls: ['./mat-table.component.css']
 })
-export class MatTableComponent implements OnInit {
+export class MatTableComponent implements OnInit,OnDestroy {
   [x: string]: any;
```
Comment (Sushmitha-Vk) [inline first_project/src/app/mat-table/mat-table.component.ts]: where us x used..?

File: first_project/src/app/mat-table/mat-table.component.ts [hunk 5]
```diff
@@ -1,43 +1,29 @@
...
+import { Component, Inject, OnDestroy, OnInit } from '@angular/core';
+import { Observable, of, Subject, takeUntil } from 'rxjs';
 import { SampleserviceService } from '../sampleservice.service';
 import { ReactiveFormComponent } from '../reactive-form/reactive-form.component';
 import { Router } from '@angular/router';
 import { MatDialog } from '@angular/material/dialog';
-
-
-
-export interface PeriodicElement {
```
Comment (Sushmitha-Vk) [inline first_project/src/app/mat-table/mat-table.component.ts:10]: where is PeriodicElement used..?

**Review by Sushmitha-Vk — CHANGES_REQUESTED** [review]

**Review by Sushmitha-Vk — APPROVED** [review]

File: first_project/src/app/mat-table/mat-table.component.ts (hunk 4 above)
Comment (Sindhujarajen) [reply first_project/src/app/mat-table/mat-table.component.ts]: resolved mam

**Review by Sushmitha-Vk — APPROVED** [review]

**Pull request merged**